### PR TITLE
fix(profiling): add installation to profiling-node

### PR DESCRIPTION
**Pull request by JonasBa**
Add install step to profiling node

### CODE REVIEW

File: src/platforms/node/common/profiling/index.mdx [hunk 1]
```diff
@@ -11,6 +11,22 @@ Node.js profiling is currently in beta, which means that it's still in progress
...
+
+```bash
+# Using yarn
+yarn add @sentry/node @sentry/tracing @sentry/profiling-node
+
+# Using npm
+npm install --save @sentry/node @sentry/tracing @sentry/profiling-node
+```
+
+## Enabling profiling
```
Comment (lizokm) [inline src/platforms/node/common/profiling/index.mdx:27]:
```suggestion
## Enabling Profiling
```

File: src/platforms/node/common/profiling/index.mdx [hunk 2]
```diff
@@ -11,6 +11,22 @@ Node.js profiling is currently in beta, which means that it's still in progress
...
+In order to enable profiling, import @sentry/profiling-node and add ProfilingIntegration to your integrations. it is important that @sentry/profiling-node is imported after @sentry/tracing.
```
Comment (lizokm) [inline src/platforms/node/common/profiling/index.mdx:28]:
```suggestion
In order to enable profiling, import @sentry/profiling-node and add ProfilingIntegration to your integrations. Make sure that @sentry/profiling-node is imported after @sentry/tracing.
```

**Review by lizokm — APPROVED** [review]
Looks good.

**Comment by JonasBa** [comment]
Thank you @lizokm 🙇🏼

**Pull request merged**